src/home_robot_spot/examples/spot_detic.ipynb

```diff
@@ -0,0 +1,114 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from home_robot.perception.detection.detic.detic_perception import DeticPerception\n",
+    "from PIL import Image\n",
+    "import numpy as np\n",
+    "from detectron2.data.detection_utils import read_image\n",
+    "from home_robot.perception.detection.detic.Detic.detic.predictor import VisualizationDemo\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "An exception occurred in telemetry logging.Disabling telemetry to prevent further exceptions.\n",
+      "Traceback (most recent call last):\n",
+      "  File \"/opt/conda/envs/home-robot/lib/python3.9/site-packages/iopath/common/file_io.py\", line 946, in __log_tmetry_keys\n",
+      "    handler.log_event()\n",
+      "  File \"/opt/conda/envs/home-robot/lib/python3.9/site-packages/iopath/common/event_logger.py\", line 97, in log_event\n",
+      "    del self._evt\n",
+      "AttributeError: _evt\n",
+      "/opt/conda/envs/home-robot/lib/python3.9/site-packages/torch/functional.py:504: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at /home/conda/feedstock_root/build_artifacts/pytorch-recipe_1673738523684/work/aten/src/ATen/native/TensorShape.cpp:3190.)\n",
+      "  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
```

Comment:
Probably want some more comments about what this notebook is for (can just add as an issue though)